### PR TITLE
Added Keyboard LCM support

### PR DESCRIPTION
**Pull request by crogers3**
To use Keyboard mode, build and run binary using sudo, and don't supply a device name:

```
sudo ./SteeringCommandDriver STEERING_DRIVER
```

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required (VERSION 3.2 FATAL_ERROR)
```
Comment (psiorx) [inline CMakeLists.txt]: Is this necessary? What are you using that needs cmake 3.2?

``` CMAKE
target_compile_features
```

?

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 project (Steering)
-add_executable(SteeringCommandDriver Joystick.cpp SteeringCommandPublisher.cpp SteeringCommandDriver.cpp)
+add_executable(SteeringCommandDriver Device.cpp Keyboard.cpp Joystick.cpp SteeringCommandPublisher.cpp SteeringCommandDriver.cpp)
 target_link_libraries(SteeringCommandDriver lcm)
+target_compile_features(SteeringCommandDriver PRIVATE cxx_range_for)
```
Comment (psiorx) [inline CMakeLists.txt]: I think you should be able to accomplish the same thing using:

``` CMAKE
set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} -std=c++11)
```

...without requiring cmake 3.2.

**Comment by crogers3** [comment]
Done

**Comment by psiorx** [comment]
:+1:

**Pull request merged**